Code/GraphMol/ChemReactions/testData/github1950_1.rxn

```diff
@@ -0,0 +1,42 @@
+$RXN
+
```

Comment:
Could you add a test with ```M  MRV``` as well in the product as well?

Reply:
Those are added